src/SDKs/DataMigration/Management.DataMigration/Microsoft.Azure.Management.DataMigration.csproj

```diff
@@ -17,6 +17,8 @@
 	</PropertyGroup>
 	<PropertyGroup>
 		<TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+		<FileVersion>0.4.0.0</FileVersion>
```

Comment:
Remove these lines
`FileVersion` must be set in the `AssemblyInfo.cs` file

Reply:
Done